autopush/web/registration.py

```diff
@@ -174,9 +174,16 @@ def validate_auth(self, data):
 
 
 def conditional_token_check(object_dict, parent_dict):
-    if parent_dict['path_kwargs']['type'] in ['gcm', 'fcm']:
+    ptype = parent_dict['path_kwargs']['type'].lower()
```

Comment:
This may sound nitpicky but I'd rather *not* be lax about case here since it's from the url path -- mostly because we're case sensitive for any other path segment we're responsible for AFAICT (and I'm picky about URIs due to my old semantic web roots). Let's not start and keep it consistent.